### PR TITLE
Fix 'hydrogeologische stratigrafie' interpretation

### PR DESCRIPTION
**Pull request by Roel**
* [x] I have read and followed the guidelines in the `CONTRIBUTING` document
* [x] I have checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change
* [x] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] All new and existing tests passed.

'Hydrogeologische stratigrafie' interpretations should only have a `pkey_boring` column because they are never linked to CPT's/sonderingen. While this was correct in the docs, we implemented both `pkey_boring` and `pkey_sondering`.

While fixing this I did some refactoring of the interpretation types, introducing two abstract base classes to deduplicate some code.

### CODE REVIEW

File: pydov/types/interpretaties.py [hunk 1]
```diff
@@ -222,28 +200,6 @@ def __init__(self):
         super(HydrogeologischeStratigrafieLaag, self).__init__(
```
Comment (pjhaest) [inline pydov/types/interpretaties.py]: Is this correct, the HydrogeologischeStratigrafieLaag in the class of InformeleStratigrafie?

**Comment by pjhaest** [comment]
There appears to be an issue with the 2.7 build of anaconda mini. 
But the new abstract classes look fine. Do you also provide a seperate base class for CPT alone? Or did I overlook that, or maybe it's not necessary?

**Comment by Roel** [comment]
Yes, I noticed the same with the Anaconda build.. It appears a very recent issue: the build of the branch was succesfull but the one of the PR is not..

Regarding your comment about HydrogeologischeStratigrafieLaag in InformeleStratigrafie (by mail), I cannot find the issue in the code (nor the comment here on Github..)?

I did not make a separate base class for CPT interpretations alone since these do not exist, so we don't need that one.

**Comment by Roel** [comment]
Interpretations never exist alone: they are either linked to a borehole or a CPT test. Some interpretation types are sometimes linked to a borehole and sometimes to a CPT test (f.ex. informele stratigrafie), others (f.ex. hydrogeologische stratigrafie) are always linked to boreholes. Interpretation types that are always linked to CPT tests do not exist.

**Comment by pjhaest** [comment]
Ok, I think I got it

**Pull request merged**